src/gfx/pal_sorting.cpp

```diff
@@ -0,0 +1,107 @@
+
+#include "gfx/pal_sorting.hpp"
+
+#include <algorithm>
+#include <png.h>
+#include <vector>
+
+#include "helpers.h"
+
+#include "gfx/convert.hpp"
+#include "gfx/main.hpp"
+
+using std::swap;
+
```

Comment:
There are no `swap` calls here.
```suggestion
using std::swap;

```